google/cloud/bigquery/schema.py

```diff
@@ -14,12 +14,16 @@
 
 """Schemas for BigQuery tables / queries."""
 
+from __future__ import annotations
 import collections
 import enum
 from typing import Any, cast, Dict, Iterable, Optional, Union
 
 from google.cloud.bigquery import _helpers
 from google.cloud.bigquery import standard_sql
+from google.cloud.bigquery._helpers import (
+    _isinstance_or_raise,
```

Comment:
Nit: Use `_helpers._isinstance_or_raise` in the code instead.

> Use import statements for packages and modules only, not for individual types, classes, or functions.

https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
This is great feedback. I had not realized that.

Reply:
Resolved.